.github/workflows/main.yml

```diff
@@ -260,3 +260,8 @@ jobs:
             echo "Error: Expected 1 entry, but got $count"
             exit 1
           fi
+
+      - name: Share to cloud
+        run: |
+          node dist/src/main.js auth login -k ${{ secrets.PROMPTFOO_STAGING_API_KEY }} -h https://api.promptfoo-staging.app
```

Comment:
maybe we should `npm link` promptfoo although that may be more ambiguous than doing it this way.

Reply:
yea, i just wanted to ensure we were running the right files